adapter/noop2/noop2.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Istio Authors.
```

Comment:
All right, based on our offline talks, the current version has a Builder and a Handler as separate interfaces. Builder is used by Mixer to do all configurations. Once the configuration is done, Builder.Build is called and from there on Mixer will just the returned Handler to do all the process calls for all the templates that the adapter expressed it wants to serve.